keyboards/mechwild/puckbuddy/info.json

```
@@ -0,0 +1,25 @@
+{
+    "keyboard_name": "MechWild PuckBuddy",
+    "url": "mechwild.com",
+    "maintainer": "Kyle McCreery",
```

This field is meant to reference your GitHub username, rather than your real name.

```suggestion
    "maintainer": "kylemccreery",
```